### DISCUSSION TITLE
Resize bar dynamically

### DISCUSSION THREAD

**Discussion by pradeepvrd** [qtile/qtile#2851] [Q&A]
I have a work environment where I frequently switch between monitors that have different dpi settings. I have a keybinding to a script that calls xrandr to adjust resolution, fix font size on my terminals and emacs instances. However I have been unable to find a way to resize the bar dynamically.

Is there a way to resize bar and widgets via the commands API?

**Comment by elParaguayo** [comment]
What issue are you having?

There is `reconfigure_screens` which should sort the bar and widgets for you. This should be automatically called if you adjust the screen with xrandr provided that your config includes `reconfigure_screens = True`.
